presto-docs/src/main/sphinx/functions/binary.rst

```diff
@@ -61,3 +61,7 @@ Binary Functions
 .. function:: xxhash64(binary) -> varbinary
 
     Computes the xxhash64 hash of ``binary``.
+
+.. function:: crc32(binary) -> varbinary
```

Comment:
Hash functions all return varbinary in Presto.

We realize that crc32 is somewhat special in this manner. And I'm suggesting that this return a bigint instead.

I agree with you that the commit message shouldn't have that note if the type is varbinary. If the type is bigint, that note is not necessary.